apps/editor.planx.uk/src/pages/FlowEditor/components/NewSettings/TeamSettings/IntegrationSettings/SubmissionEmails/index.tsx

```diff
@@ -0,0 +1,72 @@
+import Typography from "@mui/material/Typography";
+import { getIn } from "formik";
+import { useStore } from "pages/FlowEditor/lib/store";
+import React, { ChangeEvent } from "react";
+import InputLabel from "ui/editor/InputLabel";
+import Input from "ui/shared/Input/Input";
+
+import SettingsFormContainer from "../../../shared/SettingsForm";
+import {
+  GET_TEAM_SUBMISSION_INTEGRATIONS,
+  UPDATE_TEAM_SUBMISSION_INTEGRATIONS,
+} from "./queries";
+import { defaultValues, validationSchema } from "./schema";
+import {
+  GetTeamSubmissionIntegrationsData,
+  SubmissionEmailFormValues,
+  UpdateTeamSubmissionIntegrationsVariables,
+} from "./types";
+
+// Parallel feature to SubmissionEmail (singular)
+export const SubmissionEmails: React.FC = () => {
+  const teamId = useStore((state) => state.teamId);
+  console.log({ teamId });
+
+  return (
+    <SettingsFormContainer<
+      GetTeamSubmissionIntegrationsData,
+      SubmissionEmailFormValues,
+      UpdateTeamSubmissionIntegrationsVariables
+    >
```

Comment:
```suggestion
    <SettingsFormContainer<
      GetTeamSubmissionIntegrationsData,
      UpdateTeamSubmissionIntegrationsVariables
      SubmissionEmailFormValues,
    >
```

Hard to spot, but these are in the wrong order - it needs to match the following type - 

```ts
interface SettingsFormContainerProps<
  TData,
  TVariables,
  TFormValues extends FormikValues,
> {
...
}
```